tests/README.md

```diff
@@ -143,6 +143,16 @@ To run the stress test refer the Stress test [README](/tests/scripts/remote_moni
 
 To run the fault tolerant test refer the [README](/tests/scripts/remote_monitoring_tests/fault_tolerant_tests.md)
 
+### Local monitoring tests
+
+Here we test Kruize [Local monitoring APIs](/design/KruizeLocalAPI.md).
+
+#### API tests
+
+The tests does the following:
+- Deploys kruize in non-CRD mode using the deploy script from the autotune repo
+- Validates the behaviour of list datasources, import metadata and list metadata APIs in various scenarios covering both positive and negative usecases.
+
```

Comment:
Include the link to local monitoring tests doc here similar to remote monitoring